consent/manager.go

```diff
@@ -44,7 +44,7 @@ type (
 		RevokeSubjectLoginSession(ctx context.Context, user string) error
 		ConfirmLoginSession(ctx context.Context, loginSession *flow.LoginSession) error
 
-		CreateLoginRequest(ctx context.Context, req *flow.LoginRequest) (*flow.Flow, error)
+		CreateLoginRequest(ctx context.Context, req *flow.LoginRequest, f *flow.Flow) (*flow.Flow, error)
```

Comment:
```suggestion
		CreateLoginRequest(ctx context.Context, f *flow.Flow, req *flow.LoginRequest) (*flow.Flow, error)
```
it's more in line with methods below